static/app/views/dashboards/widgetCard/widgetCardChartContainer.tsx

```diff
@@ -172,6 +173,7 @@ export function WidgetCardChartContainer({
               confidence={confidence}
               sampleCount={sampleCount}
               minTableColumnWidth={minTableColumnWidth}
+              isSampled={isSampled ?? undefined}
```

Comment:
Hmm, why do we have to default to undefined ?
Isn't isSampled true, false, or undefined already?